packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -328,6 +342,11 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
     return manual ? opened : autoOpened;
   }
 
+  /** @private */
+  __computePosition(position, defaultPosition) {
+    return position || defaultPosition;
```

Comment:
Fallback to defaultPosition isn't covered by a test case

Reply:
Added tests for default position / overlay position.